deposit/src/test/java/edu/unc/lib/deposit/normalize/Simple2N3BagJobTest.java

```diff
@@ -101,18 +99,18 @@ public void depositContainerWithMODSTest() throws Exception {
 		copyTestPackage("src/test/resources/simpleMods.xml", "mods.xml", job);
```

Comment:
yes, sorry i should have stated that instead of rambling about digital object managers